enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -330,8 +330,8 @@ def test_list_view(self, role_context_dict):
                     'amount_redeemed_usd': 0.01,
                     'amount_allocated_usd_cents': 0,
                     'amount_allocated_usd': 0.00,
-                    'spend_available_usd_cents': 4,
-                    'spend_available_usd': 0.04,
+                    'spend_available_usd_cents': 0,
+                    'spend_available_usd': 0.00,
```

Comment:
This "non-redeemable" policy with a spend_limit of 0 now has no spend available.  That seems better.